Sourcery/main.swift

```diff
@@ -146,9 +146,25 @@ func runCLI() {
                 do {
                     let relativePath: Path = configPath.isDirectory ? configPath : configPath.parent()
                     configuration = try Configuration(path: yamlPath, relativePath: relativePath)
-                    Log.info("Using configuration file at '\(yamlPath)'")
+
+                    // Doesn't consider 'configPath' cause it's related to the yaml file
+                    let hasAnyParameter = (watcherEnabled == true ||
```

Comment:
Not sure I understand. In yaml file you can only specify sources/templates/outputs and extra runtime arguments, for the rest, like quite, prune and others you need to use command line arguments. I'd like to keep it like that and not clutter yaml files with booleans. So I guess here we only need to check if any sources/templates/outputs were provided with command line arguments.

Reply:
That make sense, I didn't though about those booleans, I agree that it doesn't make sense to have them in the yaml.

I'll fix it. Thanks!